test-it/test.js

```diff
@@ -249,7 +249,7 @@
         });
 
         try {
-            vireo.eggShell.loadVia(viaCode);
+            vireo.eggShell.loadVia(viaCode, {debugging: true});
```

Comment:
I had to enable debugging for both our JS and Native test suite since we do have VTR tests which check for DebugPoint execution.